openmc/data/data.py

```diff
@@ -149,6 +149,18 @@ def atomic_mass(isotope):
 
     """
     if not _ATOMIC_MASS:
+
+        # For the isotopes representing all natural isotopes of their element
+        # (e.g. C0), set atomic mass manually using the values from Atomic
+        # weights of the elements 2013 (IUPAC Technical Report)
+        # (doi:10.1515/pac-2015-0305). In cases where an atomic mass range is
+        # given (e.g. C), the average value is used.
+        _ATOMIC_MASS['c0']  = 12.0106
+        _ATOMIC_MASS['zn0'] = 65.38
+        _ATOMIC_MASS['pt0'] = 195.084
+        _ATOMIC_MASS['os0'] = 190.23
+        _ATOMIC_MASS['tl0'] = 204.3835
```

Comment:
Any reason why we shouldn't just use `atomic_mass` and `NATURAL_ABUNDANCE` to just calculate these on the fly?